.github/pull_request_template.md

```diff
@@ -1,43 +1,59 @@
 <!--
-Thank you for contributing to TiDB Operator! Please read TiDB Operator's [CONTRIBUTING](https://github.com/pingcap/tidb-operator/blob/master/CONTRIBUTING.md) document **BEFORE** filing this PR.
```

Comment:
How about still keep the link to `CONTRIBUTING.md`?

Reply:
kept